app/templates/administrative-units/administrative-unit/governing-bodies/governing-body/mandatory/new.hbs

```diff
@@ -0,0 +1,236 @@
+                    @width="block"
+                    id="contact-landline" type="tel"
+                  />
+                </:content>
+              </Item>
+              <Item @labelFor="contact-mobile">
+                <:label>GSM nummer</:label>
+                <:content>
+                  <AuInput
+                    @width="block"
+                    @value={{@model.contactMobile.telephone}}
+                    id="contact-mobile"
+                    type="tel"
+                  />
+                </:content>
+              </Item>
+            </:right>
+          </Card.Columns>
+        </:card>
+      </EditCard>
+
+      <EditCard>
+        <:title>Mandaat</:title>
+        <:card as |Card|>
+          <Card.Columns>
+            <:left as |Item|>
+              <Item @labelFor="board-position">
+                <:label>Bestuursfunctie</:label>
+                <:content>
+                  <MandateRoleSelect
+                    @selected={{@model.mandate.roleBoard}}
+                    @onChange={{fn (mut @model.mandate.roleBoard)}}
+                  />
+                </:content>
+              </Item>
+              {{!-- TODO: Implement this once we know how it should work --}}
+              {{!-- <Item @labelFor="">
+                <:label>Verkozen</:label>
+                <:content>
+                </:content>
+              </Item> --}}
+            </:left>
+            <:right as |Item|>
+              <Item @labelFor="end-date">
+                <:label>Naam eredienst</:label>
+                <:content>
+                  {{@model.administrativeUnit.name}}
+                </:content>
+              </Item>
+            </:right>
+          </Card.Columns>
+          <Card.Columns>
+            <:left as |Item|>
+              <Item @labelFor="start-date">
+                <:label>Start mandaat</:label>
+                <:content>
+                  <Datepicker
+                    @id="start-date"
+                    @onChange={{fn (mut @model.mandatory.startDate)}}
+                    @value={{@model.mandatory.startDate}}
+                  />
+                </:content>
+              </Item>
+              {{!-- TODO: Hide this if the when "lid van rechtswege" is selected --}}
+              <Item @labelFor="expected-end-date">
+                <:label>Geplande einddatum</:label>
+                <:content>
+                  <Datepicker
+                    @id="expected-end-date"
+                    @onChange={{fn (mut @model.mandatory.expectedEndDate)}}
+                    @value={{@model.mandatory.expectedEndDate}}
+                  />
+                </:content>
+              </Item>
+            </:left>
+            <:right as |Item|>
+              <Item @labelFor="governing-body-period">
+                <:label>Bestuursperiode</:label>
+                <:content>
+                  <AuInput
+                    @width="block"
```

Comment:
This should also be a disabled input because the user is adding the mandatory in the specific bestuursperiode.

Reply:
agora que vi o `disabled={{true}}`  (I'm not 100% yet 😬 )